keymanagementrequirements.md

```diff
@@ -0,0 +1,138 @@
+# Overview
+Key management for container signing can be broadly categorized into three general use cases:
+- Key Setup/Signing (Key managment and integrations with client tooling for generating signatures.)
+- Trust store configuration (Configuring which keys to trust for artifacts from a predefined source.)
+- Signature Validity (Expiry, Revocation and/or Trusted Update List. Mechanism to update information on the validity of signatures.)
+
+## Personas:
+- Publisher: User who builds and signs containers.
+    - Publisher Admin: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrator) will be responsible for configuring roots, provide access to use or generate delegate keys, and make decisions for key revocation.
+    - Publisher Builder: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Builders (i.e. developers, build hosts) will be responsible for creating artifacts. 
+    - Publisher Signer: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Signers (i.e. developers, signing hosts) will be responsible for signing artifacts.
+- Deployer: User who deploys containers.
+    - Deployer Admin: In some scenarios a deployer will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrators) will be responsible for configuring deployment policies, including potentially a set of trusted roots or required signatures. 
+    - Deployer Operator: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Operators (i.e. developers, deployment hosts) will be responsible for pulling container images from registries, verifying their signatures, and then running them.
+- Repository Owner: In some scenarios a container may be stored in a repository that is managed by the repository owner. The repository may be public or private and could also be air-gapped.
```

Comment:
There's a repo concept, and collection of repo concept where a collection of repos may live below a root repo.
This is akin to the docker hub "org" concept. 

`docker.io/org/repo`

In this case, a user may want to configure something at their org level, or the repo level.

I'm not sure we need to change anything here, but wanted to highlight this could represent a hierarchy where the repository may represent a single repo, or a collection of repos below the repo.